### ISSUE TITLE
Add markdown files based on C++ version

### ISSUE THREAD

**Issue by AnthonyCalandra** [AnthonyCalandra/modern-cpp-features#29]
```
Another suggestion would be to split the file into different .md files, such as CPP_11.md, 'CPP_14.md, andCPP_17.md`.

This way it would be a lot easier to follow each standard to study it separately without following a top to bottom concept that is a bit overwhelming in my humble opinion.

I find it confusing to start reading about the latest C++17 first and then have to scroll all the way down to the bottom to read about C++11 new features that I am more interested in.

Basically the README.md should played the role of the table of contents that lead the user to a different .md file that is fully dedicated to each C++ new-features release.

What do you think?
```
Suggestion by @stefanos82.

**Comment by stefanos82** [comment]
@AnthonyCalandra I have split the main `README.md` file into 4 different files:

- `README.md` as our table of contents / short introduction
- `CP11.md`
- `CPP14.md`
- `CPP17.md`

https://gist.github.com/stefanos82/ecf71c96204c233ed0a4fbf959d788b5

Test it and let me know mate.

Cheers.

**Comment by AnthonyCalandra** [comment]
Yep that's how I'd split it too. I'm going to keep the main readme as-is because I like having everything on one page, but I'll add the links at the top and make the separate files once in home.

**Comment by stefanos82** [comment]
No worries, take your time.

**Comment by AnthonyCalandra** [comment]
Merged. db43d4b9c630bb2caa11e555a75e4125661d8bd1
